contents/docs/contributing.md

```diff
@@ -49,10 +49,17 @@ We use [Cypress](https://www.cypress.io/) for functional frontend tests. Cypress
 
 We follow the "given, when, then" framework. Each test should provide a given (initial setup for the flow or feature being tested), when (an action performed), and then (an assertion that verifies resulting behavior). To keep our components flexible and easily maintainable within tests, we use a `data-attr` pattern where we add a page-unique identifier prop to react components for testing purposes. Then, to identify an element in a Cypress tests we use `cy.get('[dataattr=some-identifier]')`. 
 
-#### Cypress usage for local development
+### Cypress Component tests
+
+0. Run `yarn install`, `yarn global add cypress`
+1. Run `npx cypress open`. This will open cypress test runner, component tests are bundled together with source under `frontend/src`.
```

Comment:
`npx cypress open -P .` when in the project directory so that the interface will load with the project